build.gradle

```diff
@@ -35,6 +35,7 @@ plugins {
   id 'idea'
   id 'jacoco'
   id 'java-library'
+  id 'jvm-test-suite'
```

Comment:
We have introduced dependency of junit platform (https://github.com/apache/kafka/blob/trunk/gradle/dependencies.gradle#L204), so maybe we can manually declare dependencies?